src/app/routes/cpsAsset/getInitialData/index.js

```diff
@@ -52,13 +54,19 @@ const transformJson = async json => {
   }
 };
 
-export default async ({ path: pathname }) => {
+export default async ({ path: pathname, service, variant }) => {
   const { json, ...rest } = await fetchPageData(pathname);
 
+  const additionalPageData = await getAdditionalPageData(
+    json,
+    service,
+    variant,
```

Comment:
Here, we're passing the `variant`, which does not exist as an argument in `getAdditionalPageData`.

See https://github.com/bbc/simorgh/pull/6338/files#diff-e0de94df014039e48a3a35d353d4336fR31